sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/datastore/V1Beta3.java

```diff
@@ -665,27 +683,67 @@ public Write withProjectId(String projectId) {
       checkNotNull(projectId, "projectId");
       return new Write(projectId);
     }
+  }
+
+  /**
+   * A {@link PTransform} that deletes {@link Entity} objects from Cloud Datastore.
+   *
+   * @see DatastoreIO
+   */
+  public static class Delete extends DatastoreMutation {
```

Comment:
Should `Delete` take a `PCollection<Entity>` or `PCollection<Key>`? Seems more natural to take have a variant that takes just keys.